src/components/FiltersVessel/FiltersList.js

```diff
@@ -0,0 +1,138 @@
+import React, { Component } from 'react';
+import { View, ScrollView } from 'react-native'
+import { Font } from 'expo';
+
+import { FilterGroup } from './FilterGroup';
+import { filterGroup } from '../../utils/FilterGroupHelperData';
+
+import { createIconSetFromIcoMoon } from '@expo/vector-icons';
+import icoMoonConfig from '../../utils/config.json';
+import { colors } from '../../Colors';
+
+import { FiltersCheckbox } from './FiltersCheckbox';
+
+const Icon = createIconSetFromIcoMoon(icoMoonConfig, "icomoon");
+
+class FiltersList extends Component {
+
+    constructor() {
+        super();
+        this.state = {
+            fontLoaded: false,
+            activeFilterGroup: false,
+            activeFilterGroup2: false,
+            activeFilterGroup3: false,
+            activeFilterGroup4: false
+        }
```

Comment:
can you name this a little bit different? Maybe something like vesselTypeGroup, companyGroup...